actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -580,6 +580,21 @@ def stringify_keys # :nodoc:
       dup
     end
 
+    def method_missing(method_sym, *args, &block)
+      if @parameters.respond_to?(method_sym)
+        message = <<-DEPRECATE.squish
+          Method #{ method_sym } is deprecated and will be removed in Rails 5.1, as `ActionController::Parameters` no longer inherit from
```

Comment:
This is minor, but maybe wrap these lines at a reasonable length as well. They looks asymmetrical. >_>